docs/2_3_common_states.adoc

```diff
@@ -903,10 +901,10 @@ Function <<fmi3GetEventIndicators>>::
 Not allowed in Co-Simulation and Scheduled Execution.
 
 Function <<fmi3GetDirectionalDerivative>>::
-No restrictions.
+See <<fmi3GetDirectionalDerivative>>.
```

Comment:
Isn't that very confusing to read with all the linking? I think it is already confusing that the section is called "Allowed Function Calls" and there are a lot of functions listed with a remark "Not allowed ..."